wavesrv/pkg/remote/remote.go

```diff
@@ -600,7 +601,12 @@ func (msh *MShellProc) GetRemoteRuntimeState() RemoteRuntimeState {
 	if vars["remoteuser"] == "root" || vars["sudo"] == "1" {
 		vars["isroot"] = "1"
 	}
-	state.RemoteVars = vars
+	varsCopy := make(map[string]string)
+	// deep copy so that concurrent calls don't collide on this data
+	for key, value := range vars {
```

Comment:
This change has nothing to do with my pr, I found a concurrency bug

the state returned from this function gets sent to the update websocket in `NotifyRemoteUpdate` (remote.go:615)
and vars is pointing to mshell memory, so when it gets marshaled to send to the client in `RunUpdates` (scws.go:129) -> wsshell `WriteJson` (wsshell.go:75), there is a race condition where any other call to `GetRemoteRuntimeState` will cause the memory in msh.Remote.StateVars to be both written to and read at the same time, even though there is a lock